crates/moon/src/cli/coverage.rs

```diff
@@ -162,7 +162,7 @@ fn run_coverage_report_command(
     cwd: &Path,
     dry_run: bool,
 ) -> std::io::Result<std::process::ExitStatus> {
-    let mut cmd = std::process::Command::new("moon_cove_report");
+    let mut cmd = std::process::Command::new(moonutil::BINARIES.moon_cove_report.as_os_str());
```

Comment:
nit: `PathBuf` is already `AsRef<OsStr>`, so in theory this conversion is not needed (You can write `&BINARIES.moon_cove_report` and so on)